src/treeinterp/TreeProtoType.cpp

```diff
@@ -0,0 +1,80 @@
+//
+// Created by jesse on 14.10.19.
+//
+#include <include/roboteam_ai/bt/composites/Selector.hpp>
+#include "bt/RoleDivider.h"
+#include "bt/BehaviorTree.hpp"
+#include "bt/decorators/Repeater.hpp"
+#include "treeinterp/TreeProtoType.h"
+#include "bt/tactics/DefaultTactic.h"
+#include "bt/Role.h"
+#include "skills/gotopos/GoToPos.h"
+#include "skills/Attack.h"
```

Comment:
Although this does not seem like an issue now, it can be problematic if a lot of includes are in one file because it means on every little change you will have to recompile all the trees. In a testing sense that can be good, but there might be an easier way. Previously, switches also had to be recompiled pretty much every time. There might be some easier way to do it with forward declaration or templates.

Also, do we plan on having all of the trees in one file? If so, that mi